tests/rt.conf

```diff
@@ -413,13 +413,13 @@ RUN | rap_control_dyn64_phy32_debug                     | + hera hercules
 
 ### S2S tests ###
 COMPILE | s2swa | gnu | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera hercules | fv3 |
-RUN | cpld_control_p8                                   | + hera                     | baseline |
+# RUN | cpld_control_p8                                   | + hera                     | baseline |
```

Comment:
I didn't realize this previously, but it should probably be noted that these two failed tests use both aerosols and waves. The non-wave,no-aerosol test *does* run on hera/R8 and the wave+pdlib runs. This indicates to me that the issue may be aerosol related? Did anyone try to run the cpld_debug_p8 but turn off aerosols?

Reply:
I did not try this, but can give it a go now.

Reply:
I have not been able to get this to pass yet. Getting a lot of timeouts now oddly.

Reply:
even if cpld_control_noaero_p8_gnu doesn't show same OSC pt2pt issue, we will not be able to pin-point out gnu/openmpi issue. Also, it doesn't explain anything about control_wam_debug_gnu failure issue. We should revisit when new version installation of gnu is available. Of course, gocart might be sensitive with specific version of mpich or openmpi. @zach1221 I think we should move on for merging.

Reply:
I confirm cpld_control_noaero_p8_gnu is running ok on hera/rocky8.